crates/core/tedge_api/src/event.rs

```diff
@@ -166,89 +139,84 @@ mod tests {
         };
         "event parsing without text or timestamp"
     )]
+    fn parse_thin_edge_event_json(event_payload: Value, expected_event: ThinEdgeEvent) {
+        let event_type = "click_event";
+        let entity = EntityMetadata::main_device("main-device".to_string());
+        let event =
+            ThinEdgeEvent::try_from(event_type, &entity, event_payload.to_string().as_str())
+                .unwrap();
+
+        assert_eq!(event, expected_event);
+    }
+
     #[test_case(
-        "tedge/events/click_event/external_source",
         json!({
             "text": "Someone clicked",
             "time": "2021-04-23T19:00:00+05:00",
         }),
         ThinEdgeEvent {
             name: "click_event".into(),
             data: Some(ThinEdgeEventData {
-                text: Some("Someone clicked".into()),
-                time: Some(datetime!(2021-04-23 19:00:00 +05:00)),
-                extras: HashMap::new(),
-            }),
-            source: Some("external_source".into()),
+            text: Some("Someone clicked".into()),
+            time: Some(datetime!(2021-04-23 19:00:00 +05:00)),
+            extras: HashMap::new(),
+        }),
+        source: Some("external_source".into()),
```

Comment:
The previous formatting was correct. If you didn't do this intentionally by hand, there's probably something wrong with your IDE's formatter setup.

Reply:
Good catch. My IDE doesn't format the stuff inside `#[test_case]`... :/

Reply:
fixed